### PR TITLE
feat(10): (web) added start with https

### PR DESCRIPTION
**Pull request by karolinayanko**
Added `start:https` command. It'll run app on 8081 port with https protocol

### CODE REVIEW

**Review by dartwlad — APPROVED** [review]

File: packages/web/package.json [hunk 1]
```diff
@@ -3,6 +3,7 @@
   "version": "0.0.1",
   "scripts": {
     "start": "cross-env PORT=8080 react-scripts start",
+    "start:https": "cross-env HTTPS=true PORT=8081 react-scripts start",
```
Comment (dubbha) [inline packages/web/package.json]: Let's maybe use port 4330, the same one as we use on dev env on the server, just for synchronization, and not to remember too many of them?
Currently on EC2:

```
web@HTTP:
dev:    8080
stage:  8081
prod:   80

web@HTTPS:
dev:    4430
stage:  4431
prod:   443
```

I'm not saying it's perfect, but least we could repeat the same locally to lower the mental burden :)

Comment (karolinayanko) [reply packages/web/package.json]: Changed, tnx

**Review by oleksiydubovykTR — APPROVED** [review]

**Review by dubbha — APPROVED** [review]

File: packages/web/package.json [hunk 2]
```diff
@@ -3,6 +3,7 @@
   "version": "0.0.1",
   "scripts": {
     "start": "cross-env PORT=8080 react-scripts start",
+    "start:https": "cross-env HTTPS=true PORT=4430 react-scripts start",
```
Comment (dubbha) [inline packages/web/package.json:6]: Hm.. I don't think this will work without the corresponding environment change

**Review by dubbha — CHANGES_REQUESTED** [review]
Created an alternative PR that should cover our needs for both HTTPS and local API development:
https://github.com/dubbha/croton/pull/74

P.S. I'm trying to make it clear and simple, but I'm afraid this is the simplest it can get.

**Pull request closed without merging**